packages/components/src/radio/docs/migration-notes.md

```diff
@@ -1,5 +1,4 @@
-
-# Radio
+### Radio
```

Comment:
wondering why the header was changed? Are we even using this file anymore? can it be deleted instead?

Reply:
Not useful anymore but as its used in documentation page it was showing as an H1 but needed to be an h3.